A60-xds-stateful-session-affinity-weighted-clusters.md

```diff
@@ -0,0 +1,270 @@
+A60: xDS-Based Stateful Session Affinity for Weighted Clusters
+----
+* Author(s): Sanjay Pujare (@sanjaypujare)
+* Approver: markdroth
+* Status: In Review
+* Implemented in: <language, ...>
+* Last updated: 2023-03-19
+* Discussion at: 
```

Comment:
Please create the discussion thread and add a link to it here.  This starts the 10-day clock for merging the gRFC.

Reply:
done